content/en/tracing/advanced_usage/_index.md

```diff
@@ -1577,9 +1590,10 @@ Once the sampling priority has been set, it cannot be changed. This is done auto
 {{% /tab %}}
 {{% tab ".NET" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+Priority sampling is enabled by default. The default sampler automatically assigns a value of `AutoReject` or `AutoKeep` to traces, depending on their service and volume.
+
+The ability to set a trace's sampling priority manually is coming soon.
```

Comment:
Can we get rid of this line completely? We try not to promise future things in the docs, they should reflect only the current state. (exceptions for things that are in beta, as beta sorta implies coming soon.)

Reply:
Removed in d309058af05290be586bd8191ef71c9934f07823.